tests/unit/test_email.py

```diff
@@ -39,3 +39,18 @@ def test_email_job_when_mail_server_is_down(self, current_user):
 
         finally:
             requests.delete(url)
+
+    def test_email_change_password_confirmation(self, current_user):
+        """ The email is sent correctly"""
+        url = "http://mail:8025/api/v2/messages"
+        response = requests.get(url)
+        data = response.json()
+        old_total = data["total"]
+
+        send_password_change_confirm_email(current_user.email)
+
+        time.sleep(1)
```

Comment:
What's the sleep for?  Is this because we test against the Mailhog API?   Could we make this an async call rather than a sleep?